packages/react-shared-contexts/package.json

```diff
@@ -33,6 +33,7 @@
     "react-test-renderer": "^16.3.0"
```

Comment:
This package should be bumped to v9-alpha for convergence.

Reply:
Please also update config for beachball in this file:

```diff
  "beachball": {
-    "tag": "beta",
+    "tag": "alpha",
```

Reply:
This package is still a dep of `@fluentui/react`. I think we agreed to remove that dep to unblock this package's use in convergence (@khmakoto do you remember?) but it never happened.

Reply:
Can I remove that dependency in this PR? @ecraig12345

Reply:
Took the liberty and removed the dependency and modified `@fluentui/react-shared-contexts` to v9-alpha for convergence